torch/_inductor/compile_fx.py

```diff
@@ -241,6 +241,19 @@ def fake_tensor_prop(
     return fake_mode
 
 
+def get_mutating_use_stack_trace(placeholder_node) -> Optional[str]:
+    # reinplaced uses might have a single, non-copy_ use
+    if len(placeholder_node.users) == 1:
+        return next(placeholder_node.users).meta.get("stack_trace", None)
```

Comment:
oh I see. maybe add a sanity assert that `next(placeholder_node.users).target.is_mutable`? (also helpful understanding the code)

Reply:
ah, I forgot about is_mutable. will address in follow up since I need to add similar things for other skips.